sources/osgAnimation/BasicAnimationManager.js

```diff
@@ -13,14 +13,6 @@ define( [
 
     'use strict';
 
-    var createTargetID = function ( id, value ) {
-        return {
-            id: id,
-            channels: [],
-            value: value
-        };
-    };
-
     var Float = {
```

Comment:
Float is just used as operator of have operation like copy/create/init to fit into the same code like Quat/Vec3/...

float are used to animate rotate angle on an axis or any other float value